opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -371,8 +375,11 @@ message DoubleDataPoint {
   // 1970.
   fixed64 time_unix_nano = 3;
 
-  // value itself.
-  double value = 4;
+  // The value itself.
```

Comment:
Should we say that data points without a value can be considered invalid?

Reply:
Done. For the record, I think we should do a better job of documenting this protocol inside the `.proto` file, but I'd like to finish writing the `.md` format first and then copy some of the spec language back into the `.proto` later.